tests/components/lamarzocco/conftest.py

```diff
@@ -127,5 +127,6 @@ def remove_local_connection(
     """Remove the local connection."""
     data = mock_config_entry.data.copy()
     del data[CONF_HOST]
+    mock_config_entry.add_to_hass(hass)
```

Comment:
This shouldn't be needed anymore since we changed the `mock_config_entry` fixture to add the mock config entry to hass.
```suggestion
```